libs/constrained-generators/src/Constrained.hs

```diff
@@ -112,13 +111,8 @@ module Constrained (
   shrinkWithNumSpec,
   conformsToNumSpec,
   toPredsNumSpec,
-  -- TODO: this is super yucky, it would be good to implement
-  -- the linear lambda thing to get rid of this.
```

Comment:
I don't know what this means but I support the principle: the fewer super yucky things the better!